forge/src/main/java/hunternif/mc/impl/atlas/client/forge/TileTextureMapImpl.java

```diff
@@ -23,7 +23,7 @@ static public Optional<Identifier> guessFittingTextureSet(RegistryKey<Biome> bio
 
         RegistryEntry<Biome> biomeTag = MinecraftClient.getInstance().world.getRegistryManager().get(Registry.BIOME_KEY).entryOf(biome);
 
-        if (biomeTag.isIn(Tags.Biomes.IS_END)) {
+        if (biomeTag.isIn(BiomeTags.IS_END)) {
```

Comment:
I'm not sure, if that's possible now, but does Fabric and Forge now have enough equivalent BiomeTags, so the biome-guessing can be harmonized between forge and fabric?

Reply:
There's still an annoying spotty set. Things like swamps still don't have proper tags. What could be done is putting unified logic in one method, than having an impl-specific method that just returns an optional for biomes that don't have vanilla tags. LMK if you want us to give that a hand.

Reply:
I would love to see the general algorithm shared between forge and fabric and only the different tags impl-specific, but I'm unsure if that is feasible.

Reply:
Smashed up and done in 8b7f924 - the set priority is now the same between platforms, and it should be easier to see how redundant or lacking tag coverage is.